model_evaluation/Predict bias.ipynb

```diff
@@ -0,0 +1,370 @@
+      "('rnn/multi_rnn_cell/cell_0/gru_cell/candidate/bias/Adam', [256])\n",
+      "('rnn/multi_rnn_cell/cell_0/gru_cell/candidate/bias/Adam_1', [256])\n",
+      "('rnn/multi_rnn_cell/cell_0/gru_cell/candidate/kernel', [356, 256])\n",
+      "('rnn/multi_rnn_cell/cell_0/gru_cell/candidate/kernel/Adam', [356, 256])\n",
+      "('rnn/multi_rnn_cell/cell_0/gru_cell/candidate/kernel/Adam_1', [356, 256])\n",
+      "('rnn/multi_rnn_cell/cell_0/gru_cell/gates/bias', [512])\n",
+      "('rnn/multi_rnn_cell/cell_0/gru_cell/gates/bias/Adam', [512])\n",
+      "('rnn/multi_rnn_cell/cell_0/gru_cell/gates/bias/Adam_1', [512])\n",
+      "('rnn/multi_rnn_cell/cell_0/gru_cell/gates/kernel', [356, 512])\n",
+      "('rnn/multi_rnn_cell/cell_0/gru_cell/gates/kernel/Adam', [356, 512])\n",
+      "('rnn/multi_rnn_cell/cell_0/gru_cell/gates/kernel/Adam_1', [356, 512])\n",
+      "('signal_early_stopping/STOP', [])\n",
+      "('title/beta1_power', [])\n",
+      "('title/beta2_power', [])\n"
+     ]
+    }
+   ],
+   "source": [
+    "var_list = checkpoint_utils.list_variables(MODEL_DIR_OCCUPATION)\n",
+    "for v in var_list:\n",
+    "    print(v)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 99,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "kernel_occupation = np.transpose(checkpoint_utils.load_variable(MODEL_DIR_OCCUPATION, 'dense_2/kernel'))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 100,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "('beta1_power', [])\n",
+      "('beta2_power', [])\n",
+      "('dense/bias', [1])\n",
+      "('dense/kernel', [256, 1])\n",
+      "('dense_1/bias', [128])\n",
+      "('dense_1/kernel', [256, 128])\n",
+      "('embeddings', [400002, 100])\n",
+      "('final_layer/bias', [33])\n",
+      "('final_layer/bias/Adam', [33])\n",
+      "('final_layer/bias/Adam_1', [33])\n",
+      "('final_layer/kernel', [128, 33])\n",
+      "('final_layer/kernel/Adam', [128, 33])\n",
+      "('final_layer/kernel/Adam_1', [128, 33])\n",
+      "('global_step', [])\n",
+      "('rnn/multi_rnn_cell/cell_0/gru_cell/candidate/bias', [256])\n",
+      "('rnn/multi_rnn_cell/cell_0/gru_cell/candidate/kernel', [356, 256])\n",
+      "('rnn/multi_rnn_cell/cell_0/gru_cell/gates/bias', [512])\n",
+      "('rnn/multi_rnn_cell/cell_0/gru_cell/gates/kernel', [356, 512])\n",
+      "('signal_early_stopping/STOP', [])\n"
+     ]
+    }
+   ],
+   "source": [
+    "var_list = checkpoint_utils.list_variables(MODEL_DIR_GENDER)\n",
+    "for v in var_list:\n",
+    "    print(v)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 101,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "kernel_gender = np.transpose(checkpoint_utils.load_variable(MODEL_DIR_GENDER, 'final_layer/kernel'))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Compute cosinus."
```

Comment:
Nit: I think we would say "Compute Cosine" (cosinus seems to be the French term)

Reply:
Done.